include/openPMD/IO/ADIOS/ADIOS2Auxiliary.hpp

```diff
@@ -36,6 +36,219 @@
 
 namespace openPMD
 {
+/** Concrete datatype of an object available at runtime.
+ * Since ADIOS2 distinguishes unsigned char, signed char and char, this enum
+ * has all three of them (unlike the public Datatype enum)
```

Comment:
Update: let's add this to the fundamental Datatypes we support.

Also x-ref the type here in Doxygen, so it's easy to "click" through.

Reply:
`signed char` is now a fundamental supported datatype. The ADIOS2Datatype enum is no more, so the Doxygen thing is probably outdated?